css/style.scss

```diff
@@ -0,0 +1,553 @@
+@charset 'utf-8';
```

Comment:
This file should be split in multiple files to be more manageable. Let's discuss a structure in our next call. You can start by creating a "_settings.scss" partial.